apps/hash-frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/property-type-form.tsx

```diff
@@ -1,49 +0,0 @@
-import { BaseUri } from "@blockprotocol/type-system";
```

Comment:
This file has been removed and the 'check if name exists' code extracted, so it appears to be a deletion